packages/framework/ai-collab/src/test/explicit-strategy/utils.ts

```diff
@@ -76,3 +78,101 @@ export function initializeOpenAIClient(service: "openai" | "azure"): OpenAI {
 		return client;
 	}
 }
+
+/**
+ * A utility class for snapshot testing.
+ */
+export class MochaSnapshotUnitTester {
+	public constructor(
+		public readonly snapshotDirectory: string,
+		public readonly suiteName: string,
+	) {}
+
+	public expectToMatchSnapshot(
+		mochaContext: Mocha.Context,
+		output: string,
+		options?: { fileNameOverride?: string; metadata?: Record<string, string> },
+	): void {
+		const snapshotBaseFolderDir = path.join(this.snapshotDirectory, "__snapshots__");
+
+		// Create the __snapshots__ directory if it does not exist
+		if (!fs.existsSync(snapshotBaseFolderDir)) {
+			fs.mkdirSync(snapshotBaseFolderDir);
+		}
+
+		// Directory to store snapshots
+		const snapshotDir: string = path.join(
+			this.snapshotDirectory,
+			"__snapshots__",
```

Comment:
Nit: let's make this a constant inside the class so we don't have to update it in several places.

Reply:
Updated in commit 6fc87d9de1a64f7fb50d2294ce8a29b8ced9e904